app/models/service_template.rb

```diff
@@ -403,6 +399,14 @@ def provision_request(user, options = nil, request_options = {})
     result[:request]
   end
 
+  def picture=(value)
+    if value
+      super unless value.kind_of?(Hash)
```

Comment:
I like how we can choose which format we want to use and we can deprecate the other easily.  The problem with rails create_picture magic is it makes passing around hashes easier to deal with.  I think we still picture= to accept a Picture object in the future, but this can be a bridge to doing that.